admin/pages/settings/index.tsx

```diff
@@ -216,11 +218,11 @@ const Settings = () => {
           .array(z.string().min(3))
           .min(1, { message: "Tags is required" }),
         description: z.string().min(5),
-        week_days: z
-          .array(z.number().gt(0))
-          .min(1, { message: "Week days is required" }),
-        start_time: z.string().min(5),
-        end_time: z.string().min(5),
+        // week_days: z
+        //   .array(z.number().gt(0))
+        //   .min(1, { message: "Week days is required" }),
+        // start_time: z.string().min(5),
+        // end_time: z.string().min(5),
```

Comment:
Let's remove/uncomment the code.

Reply:
as we are going to add this column later and I have implemented it already and it took half day in it. so, I think we need to just comment out it for now so, we can just uncomment it later whenever we want.